models/demos/tg/llama3_70b/tt/model_config.py

```diff
@@ -197,13 +197,34 @@ def set_attention_config(model_config, max_batch_size):
         orientation=ttnn.ShardOrientation.ROW_MAJOR,
         use_height_and_width_as_shard_shape=True,
     )
+    decode_config["QKV_OUT_MEMCFG"] = lambda mesh_cols: ttnn.create_sharded_memory_config(
```

Comment:
Is this used anywhere?

Reply:
This is the memory config of the skipped ccl which has pcc issue, this will be used later when issue get resolved